install/gcp/installer/forseti_server_installer.py

```diff
@@ -232,8 +258,14 @@ def determine_access_target(self):
         """
```

Comment:
the docstring of this fn seems to be outdated now? since you allow more than org level access?

Reply:
You are correct. This should only apply to the project where forseti is being installed, not for each resource to be inventoried and scanned. As written it defaults to the Org of the first resource passed in the arg.

If this assumption is unsafe, we should set to the org of the project id passed, or the org of the project id for the running users current context, ie the org of the forseti installation project.